### PR TITLE
SAK-43492 - Web Content: When selecting the custom icon after leaving…

### PR DESCRIPTION
**Pull request by jonespm**
… focus goes to the bottom of the page

### CODE REVIEW

File: web/web-portlet/src/bundle/vm/edit.vm [hunk 1]
```diff
@@ -5,6 +5,12 @@ ${includeLatestJQuery}
 <script>
 	includeLatestJQuery('web-ifp edit.vm');
 	includeWebjarLibrary("fontawesome-iconpicker");
+	//When hiding the iconpicker focus back on the element
+	$( document ).ready(function() {
+	    jQuery("#fa_icon").on("iconpickerHide", function (event) {
```
Comment (ottenhoff) [inline web/web-portlet/src/bundle/vm/edit.vm:10]: jQuery or $ :), pick one!

Comment (jonespm) [reply web/web-portlet/src/bundle/vm/edit.vm:10]: Oops! ;)

File: web/web-portlet/src/bundle/vm/edit.vm [hunk 2]
```diff
@@ -5,6 +5,12 @@ ${includeLatestJQuery}
 <script>
 	includeLatestJQuery('web-ifp edit.vm');
 	includeWebjarLibrary("fontawesome-iconpicker");
+	//When hiding the iconpicker focus back on the element
+	$( document ).ready(function() {
```
Comment (jonespm) [inline web/web-portlet/src/bundle/vm/edit.vm:9]:
```suggestion
	jQuery(document).ready(function() {
```

File: web/web-portlet/src/bundle/vm/edit.vm (hunk 1 above)
Comment (jonespm) [reply web/web-portlet/src/bundle/vm/edit.vm:10]: I also wanted to try to get the focus of the next tabindex'able object but I wasn't sure how to do that without directly getting the id, which is probably fine in this case. Putting the focus back to where it was seemed fine though and better than it was. I'm not sure why this is working as expected in LTI, something with the HTML layout?

**Review by ottenhoff — APPROVED** [review]

**Pull request merged**